src/main/java/com/triangledeterminer/TriangleDeterminerApplication.java

```diff
@@ -2,10 +2,18 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
 
+@RestController
 @SpringBootApplication
 public class TriangleDeterminerApplication {
 
+	@RequestMapping("/")
```

Comment:
Spring Boot Application and controller mappings must be in separate classes. Stick to the SOLID principles.

Reply:
Changes done for separating controller mappings from Spring Boot Application.